src/renderer/coremods/settings/pages/Addons.tsx

```diff
@@ -125,12 +138,14 @@ function getAuthors(addon: RepluggedPlugin | RepluggedTheme): Author[] {
 }
 
 function getSourceLink(addon: RepluggedPlugin | RepluggedTheme): string | undefined {
-  const { updater } = addon.manifest;
+  const { id: addonId, updater } = addon.manifest;
   if (!updater) return undefined;
-  const { type, id } = updater;
+  const { type, id: updaterId } = updater;
   switch (type) {
     case "github":
-      return `https://github.com/${id}`;
+      return `https://github.com/${updaterId}`;
```

Comment:
If this is `manifest.updater.id`, then it'd resolve to, for example, `https://github.com/dev.alyxia.removeChannelEmojis`. Unless this value gets changed somehow in another way.

Reply:
@lexisother `updater.id` for GH addons is username/repo. https://guide.replugged.dev/docs/manifest#updatertype